### PR TITLE
update reports links to new dashboards

### LINKED ISSUE #1350: Reports Dropdown Menu inconsistent with existing summary dashboards
### Overview
The current dropdown menu of "Reports" is outdated as we combined our previous dashboards. Expected behavior is to have two items on the dropdown (Overview, Neighborhood), which links to the two new dashboards (overview_combined, nc_summary_comparison) respectively.
Dev Website

Currently, clicking on any of the dropdown would first bring to an index page (see screenshot), and clicking them in turn will bring one to the actual dashboards. @nichhk I would need some help from the engineering team to implement this change. 

### Action Items
- [ ] Remove the old reports dropdown [here](https://github.com/hackforla/311-data/blob/dev/client/components/Header.jsx#L84-L108) and replace with new links for the two new reports. The two new reports are the python files [here](https://github.com/hackforla/311-data/tree/dev/server/dash/dashboards). So the two new `Links` in Header.jsx would link to "/reports/dashboards/nc_summary_comparison" and "/reports/dashboards/overview"

### Resources/Instructions
[Code for summary dashboards](https://github.com/hackforla/311-data/tree/dev/server/dash/dashboards)

### PR DESCRIPTION
**Pull request by nichhk**
Fixes #1350

Manually verified that new reports links work as expected. Note that I had to set `REPORT_URL=https://dash-reporting.tofn9kh9mlm7g.us-east-1.cs.amazonlightsail.com
` in my client/.env file.

  - [ ] Up to date with `dev` branch
  - [ ] Branch name follows [guidelines](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md#feature-branching)
  - [ ] All PR Status checks are successful
  - [ ] Peer reviewed and approved

Any questions? See the [getting started guide](https://github.com/hackforla/311-data/blob/master/GETTING_STARTED.md)

### CODE REVIEW

File: client/components/Header.jsx [hunk 1]
```diff
@@ -81,29 +81,14 @@ const Header = () => {
...
-          </Link>
-          <Link to="/reports/dashboards/neighborhood_recent" className={classes.link}>
-            <MenuItem onClick={handleClose} button className={classes.button}>
-              Neighborhood Recent
-            </MenuItem>
-          </Link>
-          <Link to="/reports/dashboards/types_map" className={classes.link}>
-            <MenuItem onClick={handleClose} button className={classes.button}>
-              Request Type Map
+              Compare Two Neighborhoods
```
Comment (edwinjue) [inline client/components/Header.jsx:91]: may want to consider naming this "Neighborhood" for sake of brevity and as noted in #1350. longer names tend to cause the drop down to run to the right a bit as well. may want to double check with @joshuayhwu on his preference as well

**Review by edwinjue — APPROVED** [review]
Cool. Tested and works. Approving, but may first want to check with @joshuayhwu on how he feels about renaming the drop down item from "Compare Two Neighborhoods" to "Neighborhood". Thanks Nich!

File: client/components/Header.jsx (hunk 1 above)
Comment (joshuayhwu) [reply client/components/Header.jsx:91]: @edwinjue @nichhk go ahead - definitely sounds simpler and capture the essence.

**Comment by nichhk** [comment]
Thanks for the review guys!

**Comment by nichhk** [comment]
Btw @edwinjue--it's not expected that you test PRs as a reviewer. But feel free to do so if you prefer.

**Pull request merged**